File: code/c/Code_2_04_3.c
#include <stdio.h>

int main() {
	// 輸入
	int N, S;
	scanf("%d%d", &N, &S);
	
	// 求解
	int answer = 0;
	int i, j;
	for (i = 1; i <= N; i++) {
		for (j = 1; j <= N; j++) {
			if (i + j <= S) {
				answer += 1;
			}
		}
	}
	
	// 輸出
	printf("%d\n", answer);
}

File: code/python/Code_2_04_3.py
# 輸入
N, S = map(int, input().split())

# 求出答案
answer = 0
for i in range(1, N + 1):
	for j in range(1, N + 1):
		if i + j <= S:
			answer += 1

# 輸出
print(answer)
